CMakeLists.txt

```diff
@@ -1,43 +1,27 @@
+# Set the minimum version of CMake that's required 
 cmake_minimum_required(VERSION 3.12)
 
-project(maier-saupe-lc-hydrodynamics VERSION 0.1 LANGUAGES C CXX CUDA)
+project(maier-saupe-lc-hydrodynamics VERSION 0.1 LANGUAGES C CXX)
```

Comment:
I need to remove `CUDA` to get it to compile on my machine (since I don't have CUDA). You will want to put this back in.